pkg/common/configs/configs.go

```diff
@@ -33,15 +33,19 @@ const (
 	HealthCheckInterval = PrefixHealth + "checkInterval"
 
 	// events
-	CMEventTrackingEnabled    = PrefixEvent + "trackingEnabled"    // Application Tracking
-	CMEventRequestCapacity    = PrefixEvent + "requestCapacity"    // Request Capacity
-	CMEventRingBufferCapacity = PrefixEvent + "ringBufferCapacity" // Ring Buffer Capacity
+	CMEventTrackingEnabled        = PrefixEvent + "trackingEnabled"    // Application Tracking
+	CMEventRequestCapacity        = PrefixEvent + "requestCapacity"    // Request Capacity
+	CMEventRingBufferCapacity     = PrefixEvent + "ringBufferCapacity" // Ring Buffer Capacity
+	CMMaxStreamConnectionsTotal   = PrefixEvent + "maxStreamConnectionsTotal"
```

Comment:
Minor nit: This is already a pretty long var, maybe just "maxStreamConnections" (leaving out Total) and "maxStreamConnectionsPerHost". The 'Total' suffix seems redundant.